Chapters 3/Exercise_1.py

```diff
@@ -0,0 +1,4 @@
+def right_justify(s):
+    monty = ' ' * (70 - len(s)) + s
+    print(monty)
+right_justify('Monty')
```

Comment:
please configure IDE to set newline at the end of file. Then red circle will be removed here.

Reply:
added to the file .editorconfig code "insert_final_newline = true" to add a new line.